Source/Meadow.Contracts/Update/UpdateMessage.cs

```diff
@@ -18,4 +18,14 @@ public string MpakID
     /// Gets or sets the download URL of the MPak.
     /// </summary>
     public string MpakDownloadUrl { get; set; } = default!;
+    
+    /// <summary>
+    /// Gets or sets the download URL of the MPak including OS.
+    /// </summary>
+    public string MpakWithOsDownloadUrl { get; set; } = default!;
```

Comment:
What the usage for this?  We already have UpdateType in the UpdateInfo, which tells if the update has an OS, app and/or bootloader in it.

Reply:
i know we discussed on slack but replying here for posterity. the cloud will house two versions of the mpak, one with app and another with app and os. this way the device can smartly request if it needs the full or app only version based on its os version.